app/models/User.class.js

```diff
@@ -133,6 +134,10 @@ class User {
         serviceRole.service && serviceRole.service.externalId === serviceExternalId)
       .length > 0
   }
+
+  isDegatewayed () {
+    return DEGATEWAY_FLAG && this.features.includes('degatewayaccountification')
```

Comment:
There's a `hasFeature` method which does the same as `this.features.includes(...)`, only minor but could make use of it

Reply:
it's just a temporary convenience method that saves someone passing `degatewayaccountification` as a string and checking that the env var is set.